external/emulation/emulation/_emulate/microphysics.py

```diff
@@ -83,11 +129,13 @@ def microphysics(self, state: FortranState) -> None:
         # fields stay in global state so check overwrites on first step
         if self.orig_outputs is None:
             self.orig_outputs = set(state).intersection(model_outputs)
+            logger.debug(f"Overwriting existing state fields: {self.orig_outputs}")
 
-        logger.info(f"Overwritting existing state fields: {self.orig_outputs}")
         microphysics_diag = {
             f"{name}_physics_diag": state[name] for name in self.orig_outputs
         }
+
+        model_outputs.update(self.mask(state, model_outputs))
```

Comment:
Hmm. I'm not sure I follow, but I agree the naming `always_right` is a bit confusing since the user of this class doesn't care about right vs left, but rather emulator vs not. I'll change to that.

Reply:
Changed to 
```
def always_emulator(state: FortranState, emulator: FortranState):
    return emulator
```
if that helps.